examples/use_object_storage.md

```diff
@@ -35,6 +35,44 @@ upctl object-storage user create ${prefix}service --username ${prefix}user
 upctl object-storage access-key create ${prefix}service --username ${prefix}user
 ```
 
+Save the access key ID and secret access key from the output - you'll need these for S3 access.
+
+Attach a policy to grant the user access to buckets:
+
+1. **Using the UpCloud API:**
+
+   First, get your service UUID:
+   ```sh
+   upctl object-storage list
+   ```
+
+   Then attach the policy (replace `<service-uuid>` with your UUID):
+   ```sh
+   curl -X POST "https://api.upcloud.com/1.3/object-storage-2/<service-uuid>/users/${prefix}user/policies" \
+     -H "Authorization: Bearer ${UPCLOUD_TOKEN}" \
+     -H "Content-Type: application/json" \
+     -d '{"name": "ECSS3FullAccess"}'
+   ```
+
+   A successful response returns HTTP status 204.
+
+2. **Using the UpCloud Control Panel:**
+   Navigate to Object Storage → Users → Select user → Attach Policy → ECSS3FullAccess
+
+**Note:** Without attaching a policy, the user won't have permission to access buckets via AWS CLI or S3-compatible tools.
+
+Verify S3 access with AWS CLI:
+
+```sh
```

Comment:
We use these code snippets as scripts when testing `upctl` (see README in examples directory for more info), so I think this would fail because we do not have `aws` command available. Could either check if we can use curl to test the connection in somewhat nice way, or skip the code-block with `when=false`:

```suggestion
```sh when=false
```